packages/libraries/federation-link-utils/src/link-import.ts

```diff
@@ -0,0 +1,48 @@
+import { ConstValueNode, Kind } from 'graphql';
+
+export class FederatedLinkImport {
+  constructor(
+    public name: string,
+    public as: string | null,
+  ) {}
+
+  public toString(): string {
+    return this.as ? `{ name: "${this.name}", as: "${this.as}" }` : `"${this.name}"`;
+  }
+
+  static fromTypedefs(node: ConstValueNode): FederatedLinkImport[] {
```

Comment:
If see that this code (and some other in the lib implementation) is throwing errors, maybe we should use better kind of errors instead of just `Error` with a message? 🤔